src/qt/rpcconsole.cpp

```diff
@@ -725,7 +725,9 @@ void RPCConsole::updateNodeDetail(const CNodeCombinedStats *stats)
     ui->peerMinPing->setText(GUIUtil::formatPingTime(stats->nodeStats.dMinPing));
     ui->timeoffset->setText(GUIUtil::formatTimeOffset(stats->nodeStats.nTimeOffset));
     ui->peerVersion->setText(QString("%1").arg(QString::number(stats->nodeStats.nVersion)));
-    ui->peerSubversion->setText(QString::fromStdString(stats->nodeStats.strSubVer));
+    // remove leading and trailing slash
+    std::string strSubVer = stats->nodeStats.strSubVer;
+    ui->peerSubversion->setText(QString::fromStdString(strSubVer.substr(1, strSubVer.length() - 2)));
```

Comment:
Do you mind getting rid of the intermediate variable to make it parallel to the fix in the other location?

Reply:
Don't really care which way, but let's be consistent.

Reply:
Done